source/index.html.md

```diff
@@ -631,6 +631,33 @@ Replace `platerecognizer/alpr` by `platerecognizer/alpr:thailand` in the [instal
 | Tocantins           | br-to       |
 
 
+### Canada
+
+| State                     | Region Code |
+| ------------------------- | ----------- |
+| Ontario                   | ca-on       |
+| Quebec                    | ca-qc       |
+| Nova Scotia               | ca-ns       |
+| New Brunswick             | ca-nb       |
+| Manitoba                  | ca-mb       |
+| British Columbia          | ca-bc       |
+| Prince Edward Islands     | ca-pe       |
+| Saskatchewan              | ca-sk       |
+| Alberta                   | ca-ab       |
+| Newfoundland and Labrador | ca-nl       |
+
+### United Arab Emirates
+
+| State                     | Region Code |
+| ------------------------- | ----------- |
+| Abu Dhabi                 | ca-on       |
```

Comment:
`ca-on` that can't be right.

Reply:
already updated it after realizing. it should be ok now